src/preprocessing/passes/int_to_bv.cpp

```diff
@@ -69,7 +70,8 @@ Node intToBVMakeBinary(TNode n, NodeMap& cache)
     }
     else if (current.getNumChildren() > 2
              && (current.getKind() == kind::PLUS
-                 || current.getKind() == kind::MULT))
+                 || current.getKind() == kind::MULT
+                 || current.getKind() == kind::NONLINEAR_MULT))
```

Comment:
Could there be any other missing operators? If this list does not exhaust all smt-lib operators for integers, perhaps we should assert that `current.getKind()` is not any of the others?

Reply:
There are more operators (like division and mod) but we don't support them in this preprocessing pass. The check/error for unsupported operators is here: https://github.com/cvc5/cvc5/blob/2dc245a5922e660b6f44032d55d452ad27dfc20f/src/preprocessing/passes/int_to_bv.cpp#L169-L175